metacat-thrift/src/main/java/com/netflix/metacat/thrift/HiveConvertersImpl.java

```diff
@@ -352,18 +358,7 @@ public String getNameFromPartVals(final TableDto tableDto, final List<String> pa
         if (partitionKeys.size() != partVals.size()) {
             throw new IllegalArgumentException("Not the same number of partition columns and partition values");
         }
-
-        final StringBuilder builder = new StringBuilder();
-        for (int i = 0; i < partitionKeys.size(); i++) {
-            if (builder.length() > 0) {
-                builder.append('/');
-            }
-
-            builder.append(partitionKeys.get(i));
-            builder.append('=');
-            builder.append(partVals.get(i));
-        }
-        return builder.toString();
+        return FileUtils.makePartName(partitionKeys, partVals, "");
```

Comment:
I'd make this empty string a static constant so you don't have to create a new String on the heap every time.